src/other/Table.js

```diff
@@ -290,11 +305,14 @@
             })
         ;
         rows
-            .attr("class", function (d) {
+            .classed("selected", function (d) {
                 if (context._selectionBag.isSelected(context._createSelectionObject(d))) {
-                    return "selected";
+                    return true;
```

Comment:
Actually I stand corrected, the function is needed for this one because `_createSelectionObject()` needs "d".  Though I will rewrite it as 

`.classed("selected", function (d) { return context._selectionBag.isSelected(context._createSelectionObject(d))})`

It is not needed for the next one though.